### PR TITLE
Add heroku-22 stack support

### PR DESCRIPTION
**Pull request by xiexieeric**
Tested with a heroku-22 stack in my own project. Seems to work fine. Could others test it out too? 

Also added a warning regarding `playwright-chromium` version 1.15 being the last supported version

### CODE REVIEW

**Comment by xiexieeric** [comment]
@mxschmitt Pinging you for attention. First time making a PR into open source!

**Comment by brent-yearone** [comment]
I totally need this patch -- I'm hoping to run playwright on Heroku-22.

Thank you, @xiexieeric, for putting it together! 谢谢你!

**Comment by lippytak** [comment]
@xiexieeric thanks for putting this together!

Everything seems to build correctly for me, but when I try to run `playwright.chromium.launch` I still get this error:
`playwright._impl._api_types.Error: Executable doesn't exist at /app/.heroku/python/lib/python3.10/site-packages/playwright/driver/package/.local-browsers/chromium-1019/chrome-linux/chrome`

Same result with `playwright.firefox.launch`: `playwright._impl._api_types.Error: Executable doesn't exist at /app/.heroku/python/lib/python3.10/site-packages/playwright/driver/package/.local-browsers/firefox-1344/firefox/firefox
`

I'm just running this buildback and then the standard `heroku/python`.

Any idea what might be going on / suggestions? Appreciate any help.

**Comment by granth** [comment]
The browser can be installed with a `heroku-cleanup` script in `package.json`.

```javascript
"scripts": {
  "heroku-cleanup": "yarn run playwright install chromium"
}
```

**Comment by garcia-marco** [comment]
Up

**Comment by boblauer** [comment]
@xiexieeric This worked beautifully, thank you!

For anyone else who wants to try it, you can use this url for your buildpack:

https://github.com/Nitra-Finance/heroku-playwright-buildpack#518fad4074f26aeef0c1fb5d5bfaed41ae2f629e

File: README.md [hunk 1]
```diff
@@ -57,3 +57,19 @@ const { firefox } = require("playwright-firefox");
 ## Best practises
 
 It's common to only install the [browser-specific NPM packages](https://playwright.dev/#version=v1.1.1&path=docs%2Finstallation.md&q=download-single-browser-binary), which will reduce installation time and slug size on Heroku in the end, that should fix also the error that the slug size is too large.
+
+Versions of `playwright-chromium@>1.15.0` take a new dependency on `playwright-core`, which does not play well with this buildpack, causing exceptions that look like
```
Comment (mxschmitt) [inline README.md:61]: Happy to land but I think we should replace his paragraph with https://github.com/playwright-community/heroku-playwright-buildpack/pull/20#issuecomment-1370253300.

Comment (xiexieeric) [reply README.md:61]: Thanks for the review. I tried a ton of variations of `playwright install chromium` but couldn't quite get it working on my project, but I can definitely remove the comment! Excited to get this merged

Comment (xiexieeric) [reply README.md]: Comment is updated

**Review by mxschmitt — APPROVED** [review]

**Comment by jerry-yan** [comment]
Is still this being supported? Will there be heroku-24 stack support?

**Pull request merged**